test/integration/graphql_test.lua

```diff
@@ -89,14 +89,24 @@ end
 
 function g.test_variables()
     local query = [[
-        query ($arg: String! $arg2: String!) { test(arg: $arg, arg2: $arg2) }
+        query ($arg: String! $arg2: String $arg3: Int $arg4: Long $arg5: Float) {
+            test(arg: $arg, arg2: $arg2, arg3: $arg3, arg4: $arg4, arg5: $arg5)
+        }
     ]]
-    local variables = {arg = 'B', arg2 = '22'}
+    local variables = json.decode([[
+        {
+            "arg": "B",
+            "arg2": "22",
+            "arg3": 2147483647,
+            "arg4": 9223372036854775807,
+            "arg5": 9007199254740991
```

Comment:
Please extend test for float values. Bot uint64_t and int64_t should be tested. Also "negative" cases should be tested as well (when value >= 2^53)